.github/workflows/ci.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - develop
+      - develop-*
```

Comment:
Will we need this for the future or was it mostly something related to the current state of starting an instance from the `develop` branch?